new icon: discordjs (original, original-wordmark, plain, plain-wordmark)

EthanDevelops opened the pull request:
This is the first SVG I’ve ever made so there are probably a lot of things that need fixing.

From: #928 

# **New icon section**

## Double check these details before you open a PR

<!-- Tick the checkboxes to ensure you've done everything correctly -->
- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date
https://github.com/discordjs/website/blob/main/public/static/logo.svg
https://github.com/discordjs/website/blob/main/public/static/djs_logo.png
https://github.com/discordjs/discord.js/blob/main/docs/logo.svg

Panquesito7 commented on icons/discordjs/discordjs-plain.svg:
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" version="1.1">
```
Could you please provide a `plain-wordmark` version?

EthanDevelops replied:
How would I do that? (I’m new to SVG)

EthanDevelops replied:
Done! :D

Panquesito7 commented on icons/discordjs/discordjs-original-wordmark.svg:
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" >
+<defs>
+<image id="image5" width="708" height="128" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsQAAACACAYAAAAMA3/QAAAABmJLR0QA/wD/AP+gvaeTAAAgAElEQVR4nOyddXiVR9rG75lXjuWcCElIiBAI7iG4u7tDlyKl3t2t9+t2t0u3st2t69YoLS0Ud3cNDkVLcQ2QECFy9H3f+f44CVZoE3LmWPK7rnPBRZt5njMZueeZZ2YIbofAP2G+dsBLeKr+/am+CAAJAC36CACEGg0eqZSY1L+e0RhfV5RDk0VBF0WoGEupWAlECAOIBAKZgBgBaIyxPPfXYtc1puRAU64qqv2Kotovuwozf72edeTo9 ...
```
Hmm, this looks like an SVG loading an image. It'd be better if it's a good SVG. 🙂

EthanDevelops replied:
Fixed :D

Panquesito7 commented on devicon.json:
```diff
@@ -1232,6 +1232,23 @@
+            ]
+        },
+        "color": "#2a2c3e",
+        "aliases": []
```
```suggestion
        "aliases": [
          {
              "base": "original",
              "alias": "plain"
          }
      ]
```

Panquesito7 commented on devicon.json:
```diff
@@ -1232,6 +1232,23 @@
+            "nodejs"
+        ],
+        "versions": {
+            "svg": [
+                "plain",
+                "original-wordmark",
+                "plain-wordmark"
+            ]
```
```suggestion
            "svg": [
                "original",
                "original-wordmark",
                "plain",
                "plain-wordmark"
            ],
            "font": [
                "plain",
                "plain-wordmark"
            ]
```

Panquesito7 reviewed (changes requested):
Almost there! 😄

Panquesito7 commented on devicon.json:
```diff
@@ -1232,6 +1232,33 @@
+        "color": "#2a2c3e",
+        "aliases": [
+          {
+              "base": "original",
+              "alias": "plain"
```
```suggestion
              "base": "plain",
              "alias": "original"
```

Panquesito7 commented:
Looks like we have problems with the Peek Icons workflow, again 😅 could you please check, @Thomas-Boi? Thanks. 🙂

Thomas-Boi commented:
Due to the bugs related to `peek-bot` and `upload-artifact`, this PR will need longer than usual to be merged. Hopefully, it'll be merged within the next 2 weeks. Until then, thank you for your patience and understanding.

Thomas-Boi commented:
Hello @EthanDevelops,

It seems that your `plain` versions will need some work. The `plain-wordmark` should be vertically-aligned compared to the page. It also seems a bit "squished" compared to the original. When you change the SVG's viewport to 128, did you scale the text uniformly down to this size?

As for the `plain` version, I would suggest removing the outer black area (see img below). The text itself seems ok so that should do it.
![image](https://user-images.githubusercontent.com/43018778/146998733-8ba5c2ba-580c-41dd-a26a-2a7c3cffd51c.png)

All the above operations can be done using Inkscape or any SVG editor your choosing. Let me know if you need any help.

EthanDevelops commented:
> Hello @EthanDevelops,
> 
> It seems that your `plain` versions will need some work. The `plain-wordmark` should be vertically-aligned compared to the page. It also seems a bit "squished" compared to the original. When you change the SVG's viewport to 128, did you scale the text uniformly down to this size?
> 
> As for the `plain` version, I would suggest removing the outer black area (see img below). The text itself seems ok so that should do it. ![image](https://user-images.githubusercontent.com/43018778/146998733-8ba5c2ba-580c-41dd-a26a-2a7c3cffd51c.png)
> 
> All the above operations can be done using Inkscape or any SVG editor your choosing. Let me know if you need any help.

Hi,
For me on icomoon, the plain icon looks like this: 
![E3AE986D-2511-4E41-91C3-751E22784A2E](https://user-images.githubusercontent.com/66741050/147217768-2254fc06-e14f-4d8f-973d-02d35b68805c.jpeg)
Is this a bug?
Also, I couldn’t remove the outer black area because it is supposed to be a outline, as the inner area is supposed to be white.
Thanks!

EthanDevelops commented:
@Thomas-Boi

Panquesito7 commented:
> > Hello @EthanDevelops,
> > It seems that your `plain` versions will need some work. The `plain-wordmark` should be vertically-aligned compared to the page. It also seems a bit "squished" compared to the original. When you change the SVG's viewport to 128, did you scale the text uniformly down to this size?
> > As for the `plain` version, I would suggest removing the outer black area (see img below). The text itself seems ok so that should do it. ![image](https://user-images.githubusercontent.com/43018778/146998733-8ba5c2ba-580c-41dd-a26a-2a7c3cffd51c.png)
> > All the above operations can be done using Inkscape or any SVG editor your choosing. Let me know if you need any help.
> 
> Hi, For me on icomoon, the plain icon looks like this: ![E3AE986D-2511-4E41-91C3-751E22784A2E](https://user-images.githubusercontent.com/66741050/147217768-2254fc06-e14f-4d8f-973d-02d35b68805c.jpeg) Is this a bug? Also, I couldn’t remove the outer black area because it is supposed to be a outline, as the inner area is supposed to be white. Thanks!

I also uploaded it to Icomoon, and it looks like that.

![image](https://user-images.githubusercontent.com/51391473/147285875-76eff282-815a-4f5f-beb3-7e69b3d0250b.png)

Thomas-Boi commented:
So I figured out the issue: `peek-bot` needs to strip the colors off the icons. The versions you guys shown are the ones with colors. The one that the bot shows is the one *without* the color.

Example: 

Here is the *SVG* when I uploaded. As you can see on the top right, there are colors in them.
![image](https://user-images.githubusercontent.com/43018778/147298384-5f348bb6-2014-4d2f-bf9b-65c311e44db0.png)

If I were to make an icon out of the SVG, this happens:
![image](https://user-images.githubusercontent.com/43018778/147298429-ec095089-0e06-4c23-b712-28752061e16d.png)

It requires more glyphs. We only want 1 glyph per icon (see the `peek-bot`'s glyph count per icon and compare them).

How do you remove the colors? You choose Edit mode then click on the icon.
![image](https://user-images.githubusercontent.com/43018778/147298500-180aa9bf-d042-439d-b0ff-85b604b75614.png)

Then click these buttons:
![image](https://user-images.githubusercontent.com/43018778/147298511-36a0c97a-7e27-4636-8c28-a0cef200e000.png)

Now, where do we go from here? The only solution is to turn the original SVG into a one color version of itself (like [this](https://github.com/devicons/devicon/blob/master/icons/cakephp/cakephp-plain.svg) icon for example). There's only 1 color in the whole SVG (red). 

So, what you can do is turn the background transparent and make the text all white. Thus, when the color is stripped, only the black `DS.js` remains (similar to the wordmark version)

Thomas-Boi commented:
One more thing: I notice that you listed `original` as one of your SVG file but you actually don't have the file.

![image](https://user-images.githubusercontent.com/43018778/147299313-de52c11f-966b-4e1f-97e7-80c39397b2d9.png)

The `"svg"` attribute lists the SVG files that you actually have. So, you have two choices:
1. Make an `original` SVG and keep the `devicon.json`
2. Remove the attribute from the `devicon.json`

I would go with the first one. Your current `plain` SVG seems more like an `original` version to me since it has multiple colors (see above message). You should copy your `plain` version and rename it to `original`. You can then edit the `plain` version to the above tip and everything will work out.

EthanDevelops commented:
> One more thing: I notice that you listed `original` as one of your SVG file but you actually don't have the file.
> 
> ![image](https://user-images.githubusercontent.com/43018778/147299313-de52c11f-966b-4e1f-97e7-80c39397b2d9.png)
> 
> The `"svg"` attribute lists the SVG files that you actually have. So, you have two choices:
> 
> 1. Make an `original` SVG and keep the `devicon.json`
> 2. Remove the attribute from the `devicon.json`
> 
> I would go with the first one. Your current `plain` SVG seems more like an `original` version to me since it has multiple colors (see above message). You should copy your `plain` version and rename it to `original`. You can then edit the `plain` version to the above tip and everything will work out.

Hi,
Thanks! I fixed the colors and added a plain version. @Thomas-Boi

Thomas-Boi commented:
Hi @EthanDevelops,

The `plain` version still have the same issue. It looks similar to what the `original` was so it still looks strage when the color is stripped. I decided to turn the logo in to this, which should make it better. I pretty much open the SVG and remove everything except for the text in the middle. We will see how it turns out.
![image](https://user-images.githubusercontent.com/43018778/147369643-2f1f15e6-432a-4564-b36b-2beb26126a14.png)

Thomas-Boi commented on icons/discordjs/discordjs-plain.svg:
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(255,255,255);fill-opacity:1;" d="M 21.601562 38.472656 C 21.546875 38.527344 21.503906 47.203125 21.503906 57.753906 L 21.503906 76.941406 L 30.367188 76.898438 C 40.785156 76.847656 41.691406 76.75 44.75 75.324219 L 45.402344 75.019531 L 45.484375 74.117188 C 45.863281 70.042969 49.191406 66.789062 53.105469 66.671875 C 54.007812 66.644531 53. ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 0 63.9375 L 0 127.871094 L 127.871094 127.871094 L 127.871094 0 L 0 0 L 0 63.9375 M 39.742188 38.402344 C 42.941406 38.703125 46.613281 39.960938 48.621094 41.441406 C 50.507812 42.832031 52.0625 44.40625 53.003906 45.886719 C 53.839844 47.199219 53.984375 47.449219 54.074219 47.746094 C 54.128906 47.921875 54.24218 ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 21.433594 38.558594 C 21.402344 38.648438 21.390625 47.332031 21.40625 57.855469 L 21.441406 76.992188 L 21.503906 57.695312 C 21.539062 47.082031 21.550781 38.398438 21.53125 38.398438 C 21.511719 38.398438 21.46875 38.472656 21.433594 38.558594 M 37.664062 38.492188 C 37.929688 38.519531 38.359375 38.519531 38.625 ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 37.40625 38.359375 C 37.53125 38.390625 37.734375 38.390625 37.855469 38.359375 C 37.980469 38.328125 37.878906 38.300781 37.632812 38.300781 C 37.386719 38.300781 37.285156 38.328125 37.40625 38.359375 M 30.804688 46.6875 C 30.769531 46.742188 31.617188 46.769531 32.683594 46.75 C 36.21875 46.691406 36.578125 46.64 ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 25.449219 38.367188 C 27.65625 38.386719 31.226562 38.386719 33.386719 38.367188 C 35.546875 38.347656 33.742188 38.332031 29.375 38.332031 C 25.011719 38.332031 23.246094 38.347656 25.449219 38.367188 M 63.769531 38.367188 C 64.941406 38.390625 66.902344 38.390625 68.121094 38.367188 C 69.339844 38.347656 68.378906 ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 81.148438 43.230469 L 80.449219 43.96875 L 81.183594 43.269531 C 81.589844 42.882812 81.921875 42.550781 81.921875 42.53125 C 81.921875 42.4375 81.796875 42.546875 81.148438 43.230469 M 30.78125 57.761719 L 30.785156 68.671875 L 36.480469 68.679688 L 30.847656 68.601562 L 30.816406 57.726562 L 30.78125 46.847656 L 3 ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 50.9375 43.425781 C 51.253906 43.757812 51.527344 44.03125 51.546875 44.03125 C 51.644531 44.03125 51.542969 43.917969 50.976562 43.386719 L 50.367188 42.816406 L 50.9375 43.425781 M 93.984375 46.296875 C 94.105469 46.328125 94.308594 46.328125 94.433594 46.296875 C 94.554688 46.265625 94.453125 46.238281 94.207031  ...
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 53.40625 69.59375 C 53.53125 69.625 53.734375 69.625 53.855469 69.59375 C 53.980469 69.558594 53.878906 69.535156 53.632812 69.535156 C 53.386719 69.535156 53.285156 69.558594 53.40625 69.59375 M 53.152344 90.839844 C 53.273438 90.871094 53.476562 90.871094 53.601562 90.839844 C 53.722656 90.808594 53.621094 90.7812 ...
+</g>
+</svg>
```
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" version="1.1">
<g id="surface1">
<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0,0,0);fill-opacity:1;" d="M 21.433594 38.558594 C 21.402344 38.648438 21.390625 47.332031 21.40625 57.855469 L 21.441406 76.992188 L 21.503906 57.695312 C 21.539062 47.082031 21.550781 38.398438 21.53125 38.398438 C 21.511719 38.398438 21.46875 38.472656 21.433594 38.558594 M 37.664062 38.492188 C 37.929688 38.519531 38.359375 38.519531 38.625 38.492188 C 38.886719 38.464844 38.671875 38.441406 38.144531 38.441406 C 37.617188 38.441406 37.398438 38.464844 37.664062 38.492188 M 61.628906 57.023438 C 61.628906 67.300781 61.644531 71.527344 61.664062 66.410156 C 61.683594 61.292969 61.683594 52.882812 61.664062 47.722656 C 61.644531 42.558594 61.628906 46.746094 61.628906 57.023438 M 70.238281 50.078125 C 70.257812 56.503906 70.285156 64.699219 70.304688 68.289062 C 70.324219 71.878906 70.351562 66.621094 70.371094 56.609375 C 70.386719 46.59375 70.355469 38.398438 70.304688 38.398438 C 70.25 38.398438 70.222656 43.65625 70.238281 50.078125 M 92.640625 38.492188 C 92.871094 38.519531 93.21875 38.519531 93.410156 38.492188 C 93.601562 38.460938 93.414062 38.4375 92.992188 38.441406 C 92.570312 38.441406 92.414062 38.464844 92.640625 38.492188 M 94.945312 38.492188 C 95.171875 38.519531 95.546875 38.519531 95.777344 38.492188 C 96.003906 38.464844 95.816406 38.441406 95.359375 38.441406 C 94.902344 38.441406 94.714844 38.464844 94.945312 38.492188 M 81.148438 43.359375 L 80.320312 44.222656 L 81.183594 43.394531 C 81.660156 42.941406 82.046875 42.550781 82.046875 42.53125 C 82.046875 42.4375 81.917969 42.558594 81.148438 43.359375 M 93.152344 46.171875 C 93.308594 46.199219 93.570312 46.199219 93.726562 46.171875 C 93.886719 46.140625 93.757812 46.113281 93.441406 46.113281 C 93.125 46.113281 92.992188 46.140625 93.152344 46.171875 M 94.816406 46.171875 C 94.972656 46.199219 95.234375 46.199219 95.390625 46.171875 C 95.550781 46.140625 95.421875 46.113281 95.105469 46.113281 C 94.789062 46.113281 94.65625 46.140625 94.816406 46.171875 M 30.652344 46.753906 C 30.617188 46.839844 30.605469 51.835938 30.621094 57.855469 L 30.65625 68.800781 L 30.71875 57.695312 C 30.753906 51.589844 30.769531 46.59375 30.746094 46.59375 C 30.726562 46.59375 30.683594 46.664062 30.652344 46.753906 M 86.300781 51.457031 C 86.300781 51.703125 86.328125 51.804688 86.359375 51.679688 C 86.390625 51.558594 86.390625 51.355469 86.359375 51.230469 C 86.328125 51.109375 86.300781 51.210938 86.300781 51.457031 M 77.359375 52.671875 C 77.359375 53.375 77.378906 53.648438 77.40625 53.277344 C 77.433594 52.902344 77.433594 52.328125 77.40625 51.996094 C 77.378906 51.664062 77.359375 51.96875 77.359375 52.671875 M 86.300781 52.734375 C 86.300781 52.980469 86.328125 53.082031 86.359375 52.960938 C 86.390625 52.835938 86.390625 52.636719 86.359375 52.511719 C 86.328125 52.390625 86.300781 52.488281 86.300781 52.734375 M 87.800781 55.710938 C 88.117188 56.046875 88.390625 56.320312 88.410156 56.320312 C 88.507812 56.320312 88.40625 56.207031 87.839844 55.675781 L 87.230469 55.105469 L 87.800781 55.710938 M 46.132812 57.535156 C 46.132812 58.59375 46.152344 59.007812 46.175781 58.457031 C 46.199219 57.910156 46.199219 57.046875 46.175781 56.539062 C 46.152344 56.03125 46.132812 56.480469 46.132812 57.535156 M 55.855469 57.40625 C 55.855469 58.183594 55.878906 58.484375 55.902344 58.074219 C 55.929688 57.667969 55.929688 57.035156 55.902344 56.667969 C 55.875 56.300781 55.855469 56.632812 55.855469 57.40625 M 32.296875 68.832031 C 33.125 68.855469 34.453125 68.855469 35.238281 68.832031 C 36.027344 68.808594 35.347656 68.789062 33.726562 68.792969 C 32.109375 68.792969 31.464844 68.808594 32.296875 68.832031 M 53.40625 69.71875 C 53.53125 69.753906 53.734375 69.753906 53.855469 69.71875 C 53.980469 69.6875 53.878906 69.660156 53.632812 69.660156 C 53.386719 69.660156 53.285156 69.6875 53.40625 69.71875 M 45.4375 74.457031 C 45.402344 74.824219 45.421875 75.078125 45.480469 75.019531 C 45.539062 74.960938 45.566406 74.660156 45.542969 74.351562 L 45.5 73.792969 L 45.4375 74.457031 M 111.273438 75.648438 C 111.273438 76.175781 111.296875 76.390625 111.324219 76.128906 C 111.351562 75.863281 111.351562 75.433594 111.324219 75.167969 C 111.296875 74.902344 111.273438 75.121094 111.273438 75.648438 M 70.171875 75.648438 C 70.171875 75.894531 70.199219 75.996094 70.230469 75.871094 C 70.265625 75.75 70.265625 75.546875 70.230469 75.425781 C 70.199219 75.300781 70.171875 75.402344 70.171875 75.648438 M 25.449219 77.023438 C 27.582031 77.042969 31.039062 77.042969 33.128906 77.023438 C 35.21875 77.003906 33.472656 76.988281 29.246094 76.988281 C 25.023438 76.988281 23.316406 77.003906 25.449219 77.023438 M 57.019531 78.367188 L 56.511719 78.910156 L 57.054688 78.40625 C 57.5625 77.933594 57.660156 77.824219 57.5625 77.824219 C 57.542969 77.824219 57.296875 78.070312 57.019531 78.367188 M 75.136719 83.28125 C 75.136719 83.351562 75.324219 83.578125 75.550781 83.785156 L 75.96875 84.160156 L 75.585938 83.710938 C 75.371094 83.464844 75.1875 83.242188 75.167969 83.210938 C 75.152344 83.183594 75.136719 83.214844 75.136719 83.28125 M 42.492188 86.046875 C 43.019531 86.613281 43.136719 86.714844 43.136719 86.617188 C 43.136719 86.597656 42.863281 86.324219 42.527344 86.011719 L 41.921875 85.441406 L 42.492188 86.046875 M 65.40625 86.335938 C 65.097656 86.652344 64.871094 86.910156 64.90625 86.910156 C 64.941406 86.910156 65.226562 86.652344 65.535156 86.335938 C 65.847656 86.019531 66.074219 85.761719 66.039062 85.761719 C 66.003906 85.761719 65.71875 86.019531 65.40625 86.335938 M 53.152344 90.710938 C 53.273438 90.746094 53.476562 90.746094 53.601562 90.710938 C 53.722656 90.679688 53.621094 90.652344 53.375 90.652344 C 53.128906 90.652344 53.027344 90.679688 53.152344 90.710938 M 93.023438 90.714844 C 93.183594 90.746094 93.441406 90.746094 93.601562 90.714844 C 93.757812 90.683594 93.628906 90.660156 93.3125 90.660156 C 92.996094 90.660156 92.867188 90.683594 93.023438 90.714844 "/>
</g>
</svg>
```

Remove backgrounds and random paths

Panquesito7 commented on icons/discordjs/discordjs-original-wordmark.svg:
```diff
@@ -0,0 +1,16 @@
+   <path stroke="null" id="svg_10" fill="rgb(20%,70.980392%,89.803922%)" fill-rule="nonzero" d="m120.03161,73.96736c-2.16016,-0.17934 -4.19922,-1.10461 -5.78906,-2.62499l-0.23828,-0.22826l0.09766,-0.12229c0.05469,-0.06929 0.48828,-0.60326 0.95703,-1.18614l0.85938,-1.05978l0.23047,0.21196c1.14063,1.03532 2.49609,1.72011 3.86328,1.94837c0.39844,0.06522 1.26172,0.06929 1.64062,0.00815c1.16407,-0.187 ...
+  </g>
+ </g>
+</svg>
```
Missing an endline.
```suggestion
</svg>

```

Panquesito7 commented on icons/discordjs/discordjs-plain-wordmark.svg:
```diff
@@ -0,0 +1,16 @@
+   <path stroke="null" id="svg_10" fill="rgb(42,44,62)" fill-rule="nonzero" d="m120.03162,74.46909c-2.16017,-0.1644 -4.19923,-1.01257 -5.78908,-2.40624l-0.23828,-0.20924l0.09767,-0.1121c0.05467,-0.06352 0.48827,-0.55298 0.95703,-1.0873l0.85936,-0.97147l0.23047,0.1943c1.14064,0.94905 2.49611,1.57677 3.8633,1.786c0.39842,0.05978 1.2617,0.06352 1.64061,0.00748c1.16406,-0.17188 1.91016,-0.61651 2.273 ...
+  </g>
+ </g>
+</svg>
```
```suggestion
</svg>

```

Panquesito7 reviewed (approved):
LGTM 🚀 Thank you for your contribution! 😄👍
Thanks so much for your patience, and sorry for the inconvenience.

Thomas-Boi commented:
This looks good to me as well. I'll merge it now.

Thomas-Boi commented on devicon.json:
```diff
@@ -1221,6 +1221,33 @@
+        },
+        "color": "#2a2c3e",
+        "aliases": [
+          {
+              "base": "plain",
+              "alias": "original"
+          }
```
```suggestion
```
One final change: since we now have the `plain` SVG, we should just remove the alias section

Thomas-Boi reviewed (approved):
Looks good now
